app/views/conference_registrations/show.html.haml

```diff
@@ -97,33 +97,44 @@
   - if @conference.tickets.any?
     .row
       .col-md-12
+        %h4
+          %span.fa-stack
+            %i.fa.fa-square-o.fa-stack-2x
+            %i.fa.fa-ticket.fa-stack-1x
+          Tickets
         -if @tickets.any?
```

Comment:
Since we are doing all the rest, please add a space between the dash and the if

Reply:
Fixed in bedb6fad .